projects/admin/src/app/routes/documents-route.ts

```diff
@@ -74,18 +92,37 @@ export class DocumentsRoute extends BaseRoute implements RouteInterface {
             aggregations: (aggregations: any) => this._routeToolService
               .aggregationFilter(aggregations),
             aggregationsName: {
-              organisation: _('Library')
+              online: _('Online'),
+              organisation: _('Library'),
+              genreForm: _('Genre'),
+              intendedAudience: _('Audience'),
+              year: _('Publication year'),
+              subject_fiction: _('Subject (fiction)'),
+              subject_no_fiction: _('Subject (non-fiction)'),
+              contained_in: _('Contained in')
             },
             allowEmptySearch: false,
             aggregationsOrder: [
               'document_type',
-              'author',
               'organisation',
               'language',
-              'subject',
+              'year',
+              'author',
+              'subject_no_fiction',
+              'subject_fiction',
+              'genreForm',
+              'intendedAudience',
+              'contained_in',
               'status'
             ],
-            aggregationsExpand: ['document_type'],
+            aggregationsExpand: () => {
+              const expand = ['document_type'];
+              const queryParams = this._route.snapshot.queryParams;
```

Comment:
according to previous comment : `this._routeToolService.activatedRoute().snapshot.queryParams`;